llmfoundry/utils/data_prep_utils.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
nit: should we update the license to `2023` ? Applicable to all the files in this repository. May not be part of this PR but worth addressing in a separate PR.

Reply:
ah yeah, maybe, will figure this out.